tests/test_consensus_min_depth.cpp

```diff
@@ -4,10 +4,10 @@
 #include "../src/call_consensus_pileup.h"
 #include "../src/allele_functions.h"
 
-int call_cns_check_outfile(std::string prefix, std::string cns, char gap, bool call_min_depth, int min_depth){
+int call_cns_check_outfile(std::string input_id, std::string prefix, std::string cns, char gap, bool call_min_depth, int min_depth){
   std::string path = "../data/test.gap.sorted.mpileup";
   std::ifstream mplp(path);
-  call_consensus_from_plup(mplp, prefix, 20, 0, min_depth, gap, call_min_depth);
+  call_consensus_from_plup(mplp, input_id, prefix, 20, 0, min_depth, gap, call_min_depth);
   std::ifstream outFile(prefix+".fa");
   std::string l;
   getline(outFile, l);		// Ignore first line
```

Comment:
Check the name for two options,
1. `input_id` specified
2. `input_id` as `NULL`